docs/ado/guide/data/hellodata-code.md

```diff
@@ -15,7 +15,7 @@ author: rothja
 ms.author: jroth
 ---
 # HelloData Code
-```  
+```  VBA
```

Comment:
Thank you for this suggestion! I looked in our guidelines for docs, and I don't think a vba code block is supported. However, we could label this as vb and get similar formatting that is needed for the colorizer. Can I suggest that instead? 

```suggestion
```vba
```

Reply:
The external Microsoft Docs Contributor Guide doesn't include [code fencing](https://docs.microsoft.com/contribute/code-in-docs#fenced-code-blocks) for VBA, but it is listed in the internal contributor guide:

![image](https://user-images.githubusercontent.com/19416435/121056626-3b00b280-c773-11eb-8d10-3b15decb404c.png)

@rothja shall we use ```vba (no space between back ticks and letters)?